### PR TITLE
update sourcemobile + sourcedesktop

### PR DESCRIPTION
**Pull request by jadalilleboe**
Update source components to align with new skyportal changes

### CODE REVIEW

File: extensions/skyportal/static/js/components/SourceMobile.jsx [hunk 1]
```diff
@@ -292,34 +296,6 @@ const SourceMobile = WidthProvider(
                       </div>
                     </div>
                   </div>
-                  {source.duplicates && (
```
Comment (mcoughlin) [inline extensions/skyportal/static/js/components/SourceMobile.jsx:295]: @jadalilleboe Was this removed by accident perchance?

File: extensions/skyportal/static/js/components/SourceDesktop.jsx [hunk 2]
```diff
@@ -33,15 +34,13 @@ import AssignmentList from "./AssignmentList";
 import SourceNotification from "./SourceNotification";
 import EditSourceGroups from "./EditSourceGroups";
 import UpdateSourceRedshift from "./UpdateSourceRedshift";
-import SourceAnnotationButtons from "./SourceAnnotationButtons";
 import SourceRedshiftHistory from "./SourceRedshiftHistory";
 import AnnotationsTable from "./AnnotationsTable";
 import SourceSaveHistory from "./SourceSaveHistory";
 import PhotometryTable from "./PhotometryTable";
 import FavoritesButton from "./FavoritesButton";
-import TNSInfo from "./TNSInfo";
```
Comment (mcoughlin) [inline extensions/skyportal/static/js/components/SourceDesktop.jsx:42]: @jadalilleboe here as well.

**Review by mcoughlin — CHANGES_REQUESTED** [review]
Looks like a few things disappeared.

File: extensions/skyportal/static/js/components/SourceMobile.jsx (hunk 1 above)
Comment (jadalilleboe) [reply extensions/skyportal/static/js/components/SourceMobile.jsx:295]: yes I just copy pasted the component from skyportal

Comment (jadalilleboe) [reply extensions/skyportal/static/js/components/SourceMobile.jsx:295]: I didn't realize there was fritz specific stuff in there

**Comment by jadalilleboe** [comment]
@mcoughlin fixed I believe!

**Comment by mcoughlin** [comment]
@jadalilleboe https://github.com/fritz-marshal/fritz/runs/6120571673?check_suite_focus=true#step:6:2725 I think this is useNavigate or something now?

**Comment by jadalilleboe** [comment]
fixed¿ I do not have fritz set up locally.

**Review by mcoughlin — APPROVED** [review]
@jadalilleboe Thanks for fixing that. Looks good.

**Review by guynir42 — APPROVED** [review]

**Pull request merged**